### DISCUSSION TITLE
How does zigar do it?

### DISCUSSION THREAD

**Discussion by nexusprime-here** [chung-leong/zigar#439] [Q&A] [answered]
You are using FFI, or you compile zig to wasm and use on js?

**Comment by chung-leong** [comment, marked as answer]
Sorry. I wasn't checking the discussion section since no one was using it. 

node-zigar and bun-zigar compiles Zig into native code shared libraries, which are then loaded into Node.js and Bun.js through the Node API. rollup-plugin-zigar and zigar-loader compiles Zig to WASM.
